Tasks/PublishCodeCoverageResultsV1/package-lock.json

```diff
@@ -1,6 +1,842 @@
 {
+  "name": "PublishCodeCoverageResultsV1",
+  "lockfileVersion": 2,
```

Comment:
https://github.com/microsoft/azure-pipelines-tasks/pull/17277#discussion_r1032273934
I guess the same here.

Reply:
fixed